### PR TITLE
Style Guide and Diataxis Review: Wallet IV

### PR DESCRIPTION
**Pull request by charlesthompson3**
# Description of change

TW Style Guide and diataxis review has been applied to the Wallet documentation. Sections reviewed include:

- Java: API References
- Java: Examples
- Python: Getting Started
- Python: API References

## Type of change

Choose a type of change, and delete any options that are not relevant.

- Documentation Fix

## Change checklist

Tick the boxes that are relevant to your changes, and delete any items that are not.

- [x] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that new and existing unit tests pass locally with my changes

### CODE REVIEW

File: documentation/docs/libraries/python/getting_started.md [hunk 1]
```diff
@@ -28,26 +31,30 @@ You can download the latest artifacts for major python version using the  [nighl
...
 The pip list should now include the _iota-wallet-python-binding_:
+
 ```plaintext
 Package                    Version
 -------------------------- -------
 iota-wallet-python-binding 0.1.0
 ```
 
-Once you python environment has installed the `iota-wallet-python-binding` you can start developing using the python binding.
+Once your python environment has installed the `iota-wallet-python-binding` , you can start developing using the python binding.
```
Comment (Thoralf-M) [inline documentation/docs/libraries/python/getting_started.md]: Shouldn't it be
```suggestion
Once your python environment has installed the `iota-wallet-python-binding`, you can start developing using the python binding.
```
?

Comment (charlesthompson3) [reply documentation/docs/libraries/python/getting_started.md]: Sorry, VS+grammarly makes it hard with the spacing. Fixing now👌

**Comment by charlesthompson3** [comment]
@iotaledger/tech-writers

File: documentation/docs/libraries/java/examples.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-description: Official IOTA Wallet Library Java API examples.
+description: THE Official IOTA Wallet Library Java API examples.
```
Comment (lucas-tortora) [inline documentation/docs/libraries/java/examples.md]:
```suggestion
description: The Official IOTA Wallet Library Java API examples.
```

File: documentation/docs/libraries/java/examples.md [hunk 3]
```diff
@@ -21,9 +22,9 @@ git clone https://github.com/iotaledger/wallet.rs
 cd wallet.rs/bindings/java
 ```
 
-Examples are all collected in a sample project. By default it runs a node info example, but there are many more.
+Examples are all collected in a sample project. By default, it runs a node info example, but there are more.
 
-Run the example like:
+You can run the example as:
```
Comment (lucas-tortora) [inline documentation/docs/libraries/java/examples.md]:
```suggestion
You can run the example with Gradle or Maven using the following commands:
```

Comment (lucas-tortora) [reply documentation/docs/libraries/java/examples.md]: I'd lose the "Example" from the title on line 34, it's a bit redundant given that we're in the examples section.

File: documentation/docs/libraries/java/examples.md [hunk 4]
```diff
@@ -99,11 +103,13 @@ try {
 ```
 
 2. Generate the address:
+
 ```java
 Address address = account.generateAddress();
 ```
 
 3. Print and wait
```
Comment (lucas-tortora) [inline documentation/docs/libraries/java/examples.md]:
```suggestion
3. Print and wait:
```

File: documentation/docs/libraries/java/examples.md [hunk 5]
```diff
@@ -113,6 +119,7 @@ System.in.read();
 ```
 
 4. Send and wait
```
Comment (lucas-tortora) [inline documentation/docs/libraries/java/examples.md]:
```suggestion
4. Send and wait:
```

File: documentation/docs/libraries/java/examples.md [hunk 6]
```diff
@@ -127,7 +134,7 @@ Message message = account
 System.out.println("Message sent: " + message.id());
 ```
 
-## Event listening
+## Event Listening
```
Comment (lucas-tortora) [inline documentation/docs/libraries/java/examples.md]:
```suggestion
## Listen to Events
```
Alternatively, "Transfering Funds" in the previous header

File: documentation/docs/libraries/python/getting_started.md [hunk 7]
```diff
@@ -6,20 +6,23 @@ keywords:
...
 ---
 # Getting Started with IOTA Wallet Python Binding
 
 ## Security
 
-:::warning
-In a production setup, do not store passwords in the host's environment variables or in the source code. See our [backup and security recommendations](https://chrysalis.docs.iota.org/guides/backup_security) for production setups.
+:::note
+
+In a production setup, do not store passwords in the host's environment variables or in the source code. For reference, review our [backup and security recommendations](https://chrysalis.docs.iota.org/guides/backup_security) for production setups.
```
Comment (lucas-tortora) [inline documentation/docs/libraries/python/getting_started.md]: this should be a wiki link

File: documentation/docs/libraries/python/getting_started.md [hunk 8]
```diff
@@ -6,20 +6,23 @@ keywords:
...
 :::
 
 ## Installation
 
-The easiest way to get python binding up and running is to leverage pre-built python libraries for linux, MacOs or Windows that can be installed to your python environment (3.6+) via _pip_ . The binding is automagically generated using github [actions](https://github.com/iotaledger/wallet.rs/actions/workflows/python_binding_publish.yml).
+For you to get the python binding working, you need to leverage pre-built python libraries for linux, MacOs, or Windows that can be installed to your python environment (3.6+) via _pip_ . The binding is automatically generated using github [actions](https://github.com/iotaledger/wallet.rs/actions/workflows/python_binding_publish.yml).
```
Comment (lucas-tortora) [inline documentation/docs/libraries/python/getting_started.md]:
```suggestion
To get the python binding working, you need to leverage pre-built python libraries for linux, MacOs, or Windows. You can install these to your python environment (3.6+) using _pip_ . The binding is automatically generated using [GitHub actions](https://github.com/iotaledger/wallet.rs/actions/workflows/python_binding_publish.yml).
```

File: documentation/docs/libraries/python/getting_started.md [hunk 9]
```diff
@@ -28,26 +31,30 @@ You can download the latest artifacts for major python version using the  [nighl
...
 ## Usage
 
 To use the _iota_wallet_ you will need to add an import statement:  
+
 ```python
 import iota_wallet
 ```
+
 If you'd like more information on the _iota_wallet_ , you can print the documentation using the following snippet:
 ```python
```
Comment (lucas-tortora) [inline documentation/docs/libraries/python/getting_started.md:60]:
````suggestion

```python
````

**Review by lucas-tortora — APPROVED** [review]

**Review by thibault-martinez — APPROVED** [review]

**Pull request merged**